graylog2-web-interface/src/logic/notifications/NotificationsFactory.js

```diff
@@ -348,6 +374,21 @@ class NotificationsFactory {
             </span>
           ),
         };
+      }
+
+      case 'archiving_summary':
+        return {
+          title: 'The following indices could not be archived yet (but will still be retried)',
```

Comment:
The new message reads better but also leaves some questions open: Why could those indices not be archived? How many times will Graylog try? Did I lose any messages?

I think we should keep the title short, so I would vote for doing the following:
1. Keep `The following indices could not be archived yet`, or maybe `The following indices could not be automatically archived yet`, to emphasise that this happened in the background
2. Add more information in the description, something like:

```js
<span>
There was an error while archiving some indices during the index rotation process. Graylog will continue trying to archive those indices and will keep all indices until they are successfully archived.
<br />
Please check the following error messages as your assistance (or from a Graylog account manager) may be necessary to fix the issue:
<br />
<ul>(...)</ul>
</span>
```

What do you think?